hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -276,5 +276,11 @@ module clkmgr import clkmgr_pkg::*; (
   // Assertions
   ////////////////////////////////////////////////////
 
+  `ASSERT_KNOWN(TlDValidKnownO_A,  tl_o.d_valid     )
+  `ASSERT_KNOWN(TlAReadyKnownO_A,  tl_o.a_ready     )
```

Comment:
nit: can we delete all unneeded whitespace here to make alignment more consistent after generation?

Reply:
updated.